build/scons-xcode-project-build.js

```diff
@@ -49,59 +45,17 @@ async function generateIndexJSON(dirToTraverse) {
 	return fs.writeFile(destFile, JSON.stringify(index));
 }
 
-function determineBabelOptions() {
-	// eslint-disable-next-line security/detect-non-literal-require
-	const { minIosVersion } = require(path.join(ROOT_DIR, 'iphone/package.json'));
-	const options = {
-		targets: {
-			ios: minIosVersion
-		},
-		useBuiltIns: 'entry',
-		// DO NOT include web polyfills!
-		exclude: [ 'web.dom.iterable', 'web.immediate', 'web.timers' ]
-	};
-	return {
-		presets: [ [ '@babel/env', options ] ],
-		exclude: 'node_modules/**'
-	};
-}
-
-async function copyPolyfills(dir) {
-	console.log('Copying JS polyfills over');
-	const modulesDir = path.join(dir, 'node_modules');
-	// make sure our 'node_modules' destination directory exists
-	await fs.emptyDir(modulesDir);
-	copyPackageAndDependencies('@babel/polyfill', modulesDir);
-}
+async function generateBundle(outputDir) {
+	const builder = new Builder({ args: [ 'ios' ] });
+	const ios = new IOS({ });
 
-async function generateBundle(inputDir, outputDir) {
-	console.log('Transpiling and bundling common SDK JS');
-
-	// create a bundle
-	console.log('running rollup');
-	const babelOptions = determineBabelOptions();
-	const bundle = await rollup({
-		input: `${inputDir}/ti.main.js`,
-		plugins: [
-			resolve(),
-			commonjs({
-				ignore: [ '/semantic.colors.json' ]
-			}),
-			babel(babelOptions)
-		],
-		external: [ './app', 'com.appcelerator.aca', '/semantic.colors.json' ]
-	});
+	await builder.transpile('ios', ios.babelOptions, path.join(outputDir, 'ti.main.js'));
 
-	// write the bundle to disk
-	console.log('Writing common SDK JS bundle to disk');
-	return bundle.write({ format: 'cjs', file: `${outputDir}/ti.main.js` });
 }
 // FIXME: Combine common code here and in packager.js!
```

Comment:
Looks like maybe this can removed, since that's what you're doing here... :)

Reply:
Doh 😆